components/expenses/ProcessExpenseButtons.js

```diff
@@ -216,13 +214,6 @@ const ProcessExpenseButtons = ({
           error={error}
         />
       )}
-      {permissions.canMarkAsIncomplete && props.displayMarkAsIncomplete && (
```

Comment:
Should we remove https://github.com/opencollective/opencollective-frontend/blob/2d8feedda043ab1e635b70c160e11b52fcd33a36/components/expenses/ProcessExpenseButtons.js#L62 as well if we're getting rid of this?